clang/include/clang/Sema/SemaSPIRV.h

```diff
@@ -20,7 +20,7 @@ namespace clang {
 class SemaSPIRV : public SemaBase {
 public:
   SemaSPIRV(Sema &S);
-
+  bool CheckVectorArgs(CallExpr *TheCall, unsigned NumArgsToCheck);
```

Comment:
This is a helper, we don't need to expose it in the class. No one needs to use this outside of spirv and if they did it wouldn't live in the semaSPIRV file it would live somewhere like SemaChecking.